packages/web-react/src/types/radioField.ts

```diff
@@ -6,6 +6,8 @@ interface InputProps extends InputBase, ValueBase<string | number> {}
 export interface RadioFieldProps extends ChildrenProps, StyleProps, LabelProps, InputProps, TransferProps {
   /** Whether the checkbox is checked */
   isChecked?: boolean;
+  /** Whether the checkbox is an Item */
+  isItem?: boolean;
```

Comment:
Please, create `ItemProps` and use it for extension the same way as `LabelProps` and others.